autobotAI_integrations/integrations/google_chat/webhook_client.py

```
@@ -57,5 +72,11 @@ def send(self, title, body, buttons=[]):
             data=byte_encoded,
         )
         req.add_header("Content-Type", "application/json; charset=UTF-8")
-        resp = request.urlopen(req)
-        return resp
+        try:
+            resp = request.urlopen(req)
```

Ruff (0.15.13)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)